test/evp_extra_test.c

```diff
@@ -941,5 +981,8 @@ int setup_tests(void)
     if (!TEST_int_eq(EVP_PKEY_meth_add0(custom_pmeth), 1))
         return 0;
     ADD_ALL_TESTS(test_EVP_PKEY_check, OSSL_NELEM(keycheckdata));
+#ifndef OPENSSL_NO_POLY1305
+    ADD_TEST(test_EVP_poly1305);
+#endif
```

Comment:
I'm working on changes in `test/evp_test.c` to become more generic, so you should only have to change `test/recipes/30-test_evp_data/evpmac.txt`.

Reply:
Right, that is the right way of testing an EVP interface